src/renderer/components/SettingsPage.tsx

```diff
@@ -604,30 +609,33 @@ export default class SettingsPage extends React.PureComponent<Record<string, nev
                     style={{marginLeft: '20px'}}
                 >
                     {'Icon theme: '}
-                    <Radio
+                    <FormCheck
+                        type='radio'
                         inline={true}
                         name='trayIconTheme'
                         value='light'
                         defaultChecked={this.state.trayIconTheme === 'light' || !this.state.trayIconTheme}
                         onChange={() => this.handleChangeTrayIconTheme('light')}
-                    >
-                        {'Light'}
-                    </Radio>
+                        label={'Light'}
+                    />
                     {' '}
-                    <Radio
+                    <FormCheck
+                        type='radio'
                         inline={true}
                         name='trayIconTheme'
                         value='dark'
                         defaultChecked={this.state.trayIconTheme === 'dark'}
                         onChange={() => this.handleChangeTrayIconTheme('dark')}
-                    >{'Dark'}</Radio>
+                        label={'Light'}
```

Comment:
This was previously 'Dark' before, is the change intentional?

Reply:
Oops, nope